common/test/Configuration.IoTHub.cs

```diff
@@ -27,6 +27,8 @@ public static X509Certificate2 GetCertificateWithPrivateKey()
 
             public static string DeviceConnectionString => GetValue("IOTHUB_DEVICE_CONN_STRING");
 
+            public static string DeviceConnectionString2 => GetValue("IOTHUB_DEVICE_CONN_STRING2");
```

Comment:
We should set the default value as Empty (since these are used only by samples and perf tests).

Reply:
I need this to be set in order to run the tests. If we set the default value, it could create issues when the perf test runs.
No exceptions are going to be thrown until it's used. E2E tests don't use it yet.